lerobot/scripts/visualize_dataset.py

```diff
@@ -117,7 +120,27 @@ def visualize_dataset(
         ), "Set an output directory where to write .rrd files with `--output-dir path/to/directory`."
 
     logging.info("Loading dataset")
-    dataset = LeRobotDataset(repo_id, root=root)
+
+    if repo_id:
+        dataset = LeRobotDataset(repo_id, root=root)
+    elif hdf5_data_dir:
+        data_dir = Path(hdf5_data_dir)
+        if data_dir.exists():
+            logging.info(f"HDF5 data dir: {data_dir}")
+            dataset, episode_data_index, info = from_raw_to_lerobot_format(raw_dir=data_dir, videos_dir=None, fps=5, video=False)
+            stats = compute_stats(dataset, batch_size=16, num_workers=16, max_num_samples=None)
```

Comment:
@hemanthsarabu I'm thinking we don't actually need to do compute_states. but im not too sure